tools/android/packaging/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,6 @@
+#Fri Sep 01 12:45:21 CEST 2017
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+zipStoreBase=GRADLE_USER_HOME
+zipStorePath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-3.3-bin.zip
```

Comment:
do we really need to include gradle-wrapper.jar and it's properties or can we use it from the SDK?

Reply:
Good question as well.
The gradle in SDK is antiquated, though
```
distributionUrl=http\://services.gradle.org/distributions/gradle-1.12-all.zip
```

Reply:
The fact that this gradle wrapper in in a "template" subdir suggests it's not supposed to be used directly, imo

Reply:
ok